azure/nat-gateway/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_public_ip" "natgw" {
+  name                = "${var.project}-${var.environment}-ip"
```

Comment:
why the "ip" prefix at the end?

Reply:
This creates a public associated with the nat gateway. My idea was to have an identification instead of having 2 different resources with the same name